app/src/main/res/layout/group_making_settings.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:background="@color/white"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    >
+
```

Comment:
As I was thinking about this feature in my head, I realized that the number of names currently in the list would be helpful to the user to help figure out proper group settings. So let's put a TextView at the top that says, "This list has X names." where X is the number of names in the list. textSize of 17dp and textColor of dark_gray should be good...?